libavutil/hwcontext_vulkan.c

```diff
@@ -1659,6 +1659,88 @@ static void vulkan_frame_free(void *opaque, uint8_t *data)
     av_free(f);
 }
 
+static int alloc_bind_single_mem(AVHWFramesContext *hwfc, AVVkFrame *f,
```

Comment:
nearly same code as alloc_bind_mem, could we pass one flag to 
static int alloc_bind_mem(AVHWFramesContext *hwfc, AVVkFrame *f,    void *alloc_pnext, size_t alloc_pnext_stride, int use_single_linear_images )

Reply:
I've merged them into one function.